torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1492,6 +1492,15 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                        SkipInfo('TestUnaryUfuncs'),
                    ),
                    sample_inputs_func=sample_inputs_clamp),
+    UnaryUfuncInfo('conj',
+                   ref=np.conj,
+                   dtypes=all_types_and_complex_and(torch.bool,
+                                                    torch.bfloat16, torch.half),
+                   dtypesIfCPU=None,
```

Comment:
why are `dtypesIf{CPU, CUDA, ROCM}` set to `None`?

Reply:
https://github.com/pytorch/pytorch/blob/c4a8f0ceaa24f3922b1e0d96c75bb9c2af9c7e4a/torch/testing/_internal/common_methods_invocations.py#L178-L182

However since it is UnaryUfuncInfo we have to set all of them to None or all of them to
`all_types_and_complex_and(torch.bool, torch.bfloat16, torch.half)` as the UnaryUfuncInfo constructor defaults those values in its constructor

https://github.com/pytorch/pytorch/blob/c4a8f0ceaa24f3922b1e0d96c75bb9c2af9c7e4a/torch/testing/_internal/common_methods_invocations.py#L315-L319

Reply:
ohhh okay!